mainapp/templates/mainapp/disclaimer.html

```diff
@@ -7,6 +7,6 @@ <h1 class="text-center">Disclaimer</h1>
     <p style="font-size:1.7rem;">This portal has been developed for effective collaboration between various stakeholders such as citizen, Government authorities, various NGOs and volunteers. IEEE makes no representation regarding the completeness, accuracy of any information and data contained on this portal or that such information and data will be error-free. Stakeholders are trying to make their best effort to support, any information received through the portal the requests received through this portal doesn’t warrant any commitment on the services. By using this site you acknowledge that you have read and agree to this</p>
     <br>
     <h1 class="text-center">GDPR Compliance</h1>
-    <p style="font-size:1.7rem;">By providing your personal information, you consent to making such information accessible to persons who visit this website. You further acknowledge and agree that in order to serve the specific purposes for which personal information is volunteered by you, such information needs to be accessible to the public at large and therefore, by volunteering your information on this website, you consent to making such information public. By providing your phone number and email address, you further consent to being contacted by anyone who accesses such information through this website. The creators of this website or the government of Kerala shall not be liable for any unauthorized use, misuse or sharing of such information.</p>
+    <p style="font-size:1.7rem;">By providing your personal information, you consent to making such information accessible to persons who visit this website. You further acknowledge and agree that in order to serve the specific purposes for which information is volunteered by you, such information needs to be accessible to the public at large and therefore, by volunteering your information on this website, you consent to making such information publicly accessible. Further, by providing your phone number and email address, you consent to being contacted by anyone who accesses such information through this website.</p>
```

Comment:
Fixing grammatical errors, how about this?

By providing your personal information, you consent to make that information accessible to people who visit this website. You further acknowledge and agree that to serve the specific goals of the site, the information volunteered by you needs to be accessible to the public and therefore, by volunteering your information on this website, you consent to make your information publicly available. Moreover, by providing your phone number and email address, you agree to be contacted by anyone who accesses such information through this website